GOVERNANCE.md

```diff
@@ -156,40 +187,41 @@ backgrounds, viewpoints and talents in our team, which is why we explicitly do
 not define code as the sole metric on which council membership will be
 evaluated.
 
-If a Council member becomes inactive in the project for a period of one year,
+Council membership is assigned for a two year period, with no limit on how many
+periods can be served.
+
+Council members can renounce at any time and are
+encouraged to do so if they foresee they won't be able to attend their
+responsibilities for an extended interval of time.
+
+If a Council member becomes inactive in the project for a period of six months,
 they will be considered for removal from the Council. Before removal, inactive
 Member will be approached by the BDFL to see if they plan on returning to
 active participation. If not they will be removed immediately upon a Council
 vote. If they plan on returning to active participation soon, they will be
-given a grace period of one year. If they don’t return to active participation
+given a grace period of six months. If they don’t return to active participation
 within that time period they will be removed by vote of the Council without
 further grace period. All former Council members can be considered for
-membership again at any time in the future, like any other Project Contributor.
+membership again at any time in the future, like any other Core Contributor.
  Retired Council members will be listed on the project website, acknowledging
 the period during which they were active in the Council.
 
 The Council reserves the right to eject current Members, other than the BDFL,
 if they are deemed to be actively harmful to the project’s well-being, and
 attempts at communication and conflict resolution have failed.
 
-### Conflict of interest
+<!-- todos:
 
-It is expected that the BDFL and Council Members will be employed at a wide
-range of companies, universities and non-profit organizations. Because of this,
-it is possible that Members will have conflict of interests. Such conflict of
-interests include, but are not limited to:
+- vote of no conficence. Either core or recurring contributors
+should also be able to remove people from the council, even if by a very high
+majority vote. IMO (Oriol) this point is not up for discussion in itself, only in
+how it should happen.
```

Comment:
* How is such a vote initiated?
* What is the voting mechanism?
  * Attendance meeting vs. asynchronous
  * Minimum number of non-neutral votes (or similar)
  * simple/50/66/75... % majority required?

Reply:
We can copy that from ArviZ: https://github.com/arviz-devs/arviz/blob/main/GOVERNANCE.md#vote-of-no-confidence

Reply:
The ArviZ section on votes of no confidence sounds good to me, with one exception: if a vote of no confidence for a single Council Member is called by Council Members, who then decide to ask the whole Core Community to vote, it seems to me still a two thirds majority would be needed. As it's phrased now, in that case, only a simple (half) majority is needed.

(Example: let's say 3/10 Council Members want one Council Member removed (but 7/10 do not), if those three Council Members then call the full Core Team to vote, can they bypass the needed two thirds majority typically required for votes of no confidence? Seems like a loophole we do not want. Very unlikely of course, but easy to fix.)